### PR TITLE
[NT-1903] Updating copy for comments files

### PR DESCRIPTION
**Pull request by singhhari**
# 📲 What

This PR is updating some static English placeholder copy we originally put in place in aid in development of the UI/UX. The copy was finalized and translated to support the many languages our app supports.

# 🤔 Why

Kickstarter is available in many non-English speaking countries and it is our responsibility to provide a localized experience for our users.

# 🛠 How

There are always a few different steps involved in translating copy. First off, we add the English strings into a config file on the web repo. Those strings are then submitted to CrowdIn, where they are manually reviewed by a translator. After approval, on the next web deployment, they are available to be pulled down on native.

# 👀 See

It would be best to refer to the screenshot recordings in the `Files Changed` section of the PR.

# ✅ Acceptance criteria

- [ ] Switch your device language and navigate to the CommentsViewController. Verify that the copy is not rendering in English.

# ⏰ TODO

- [ ] One specific Spanish string took longer than the others to translate, so we need to wait on the next web deployment before that will be available.

### CODE REVIEW

File: Kickstarter-iOS/Locales/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -121,6 +121,7 @@
 "Continue_with_quantity_count_add_ons.two" = "Continue with %{quantity_count} add-ons";
 "Continue_with_quantity_count_add_ons.zero" = "Continue with %{quantity_count} add-ons";
 "Continue_with_this_reward" = "Continue with this reward?";
+"Couldn't_post" = "Couldn't post. Tap to retry";
```
Comment (singhhari) [inline Kickstarter-iOS/Locales/Base.lproj/Localizable.strings:124]: Is it OK for this apostrophe to be here? The function generated in the `Strings` file required a manual removal of it before compiling.

Comment (singhhari) [reply Kickstarter-iOS/Locales/Base.lproj/Localizable.strings:124]: Nvm, going to remove this apostrophe and update.

File: Kickstarter-iOS/Views/Cells/CommentCell.swift [hunk 2]
```diff
@@ -112,12 +114,10 @@ final class CommentCell: UITableViewCell, ValueCell {
 
 // MARK: Styles
 
-// TODO: Internationalized in the near future.
-
 private let replyButtonStyle: ButtonStyle = { button in
   button
     |> UIButton.lens
-    .title(for: .normal) %~ { _ in localizedString(key: "Reply", defaultValue: "Reply") }
+    .title(for: .normal) %~ { _ in Strings.general_navigation_buttons_reply() }
```
Comment (singhhari) [inline Kickstarter-iOS/Views/Cells/CommentCell.swift:120]: Used an existing string for `Reply`. This should be fine?

File: Kickstarter-iOS/Locales/Base.lproj/Localizable.strings (hunk 1 above)
Comment (singhhari) [reply Kickstarter-iOS/Locales/Base.lproj/Localizable.strings]: [This PR](https://github.com/kickstarter/kickstarter/pull/21788) should handle the issue of `make strings` returning the string with the apostrophe again.

**Review by msadoon — APPROVED** [review]
Good to go!

**Pull request merged**